src/ray/gcs/gcs_server/gcs_server.h

```diff
@@ -207,8 +208,15 @@ class GcsServer {
   /// Stats handler and service.
   std::unique_ptr<rpc::StatsHandler> stats_handler_;
   std::unique_ptr<rpc::StatsGrpcService> stats_service_;
-  // Synchronization service for ray.
-  std::unique_ptr<gcs_syncer::RaySyncer> ray_syncer_;
+  /// Synchronization service for ray.
+  std::unique_ptr<gcs_syncer::RaySyncer> gcs_ray_syncer_;
+
+  /// Ray Syncer realted fields.
+  std::unique_ptr<syncer::RaySyncer> ray_syncer_;
```

Comment:
what's the difference between gcs_ray_syncer_ vs ray_syncer_? it's not clear from reading the code.